src/main/kotlin/no/nav/tilleggsstonader/sak/vilkår/vilkårperiode/domain/MålgruppeType.kt

```diff
@@ -37,10 +37,18 @@ enum class MålgruppeType(
         gyldigeAktiviter = emptySet(),
         faktiskMålgruppe = null,
     ),
+    TILTAKSPENGER(
+        gyldigeAktiviter = setOf(AktivitetType.TILTAK),
+        faktiskMålgruppe = null,
+    ),
     INGEN_MÅLGRUPPE(
         gyldigeAktiviter = emptySet(),
         faktiskMålgruppe = null,
     ),
+    KVALIFISERINGSSTØNAD(
+        gyldigeAktiviter = emptySet(),
```

Comment:
```suggestion
        gyldigeAktiviter = gyldigeAktiviter = setOf(AktivitetType.TILTAK),
```
Kombinasjonen tiltak og kvalifiseringsstønad er fyldig